babel.config.js

```diff
@@ -2,6 +2,13 @@ module.exports = function(api) {
   api.cache(true);
   return {
     presets: ['babel-preset-expo'],
-    plugins: ['react-native-reanimated/plugin'],
+    plugins: [
```

Comment:
I don't think we need animations anytime soon?

Reply:
Default w/ starter template. Will remove once we remove usage in codebase.